### PR TITLE
Fixes some tiny typos and inconsistencies in Rust documentation

### PR DESCRIPTION
**Pull request by behradkhodayar**
Thanks for all the great effort put into bringing this valuable document into life.

Just noticed some minor typos.

This PR fixes those.

### CODE REVIEW

File: docs/Rust/section8/intro.md [hunk 1]
```diff
@@ -8,6 +8,6 @@ description: An introduction to using Cargo, Rust's Package Managment System.
 This next module will focus on understanding the structure of a Rust project.  It will be less in the playground environment we've been using and more in a code editor like Visual Studio Code and the terminal.
 
 - [Defining & Reading cargo.toml](./defining-cargo-config.md)
-- [Installing a crate](./installing-crate.md)
```
Comment (CrackTheCode016) [inline docs/Rust/section8/intro.md:11]: Please keep this order but change `sidebars.js` (located [here](https://github.com/behradkhodayar/w3f-education/blob/main/sidebars.js#LL269C8-L269C8)) to match this layout, as I changed the order slightly when I recorded the videos.  The layout should be:

- Defining & Reading cargo.toml
- Installing a crate
- Defining features for a crate
- Unit Tests in Rust

Again, thank you so much for this!

**Review by CrackTheCode016 — CHANGES_REQUESTED** [review]
@behradkhodayar Thank you very much for this! I really appreciate it, and is always open for more feedback as it materializes further.  

Just make that change and I am happy to merge it.

File: docs/Rust/section8/intro.md (hunk 1 above)
Comment (behradkhodayar) [reply docs/Rust/section8/intro.md]: Thanks for the prompt preview & response. Applied the changes [which of course I believe makes more sense contextually compared to the previous one].

Guess its ready.

Looking forward to great upcoming material :-)

**Review by CrackTheCode016 — APPROVED** [review]
@behradkhodayar thank you very much! 🎉

**Comment by behradkhodayar** [comment]
I appreciate you assessing & merging quickly during the weekend🙏

**Pull request merged**